applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp

```diff
@@ -310,17 +297,17 @@ namespace Kratos
 		const Matrix& Ncontainer = this->GetGeometry().ShapeFunctionsValues(
 			GeometryData::GI_GAUSS_1);
 
-		bounded_vector<double,3> EquivalentLineLoad = ZeroVector(3); 
-		bounded_vector<double,msElementSize> BodyForcesGlobal = ZeroVector(msElementSize);
+		bounded_vector<double,3> equivalent_line_load = ZeroVector(3); 
+		bounded_vector<double,msElementSize> body_forces_global = ZeroVector(msElementSize);

```

Comment:
you are completely overwriting ths vector on line 322. You don't need the zero initialization

Reply:
I am not writing values to the 3rd and 6th entry so I need the zero init.